generator/src/main/java/dev/cdevents/generator/CDEventsGenerator.java

```diff
@@ -68,7 +68,11 @@ public static void main(String[] args) {
                     SchemaData schemaData = buildCDEventDataFromJsonSchema(file);
                     generateClassFileFromSchemaData(mustache, schemaData, targetPackageDir);
                 }
+            } else {
+                log.error("No schema files found in the specified directory {}", folder.getAbsolutePath());
             }
+        } else {
+            log.error("No schema directory found in the specified directory {}", folder.getAbsolutePath());
```

Comment:
For automation purposes, it would be nice if the tool accumulated the various errors and failed at the end, rather than only logging errors.